src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -882,37 +882,39 @@ private Task WriteSuffix()
 
         private void CreateResponseHeader(bool appCompleted)
         {
+            var keepAlive = _keepAlive;
```

Comment:
Add comments explaining the reason for the locals, otherwise they look redundant.